### PR TITLE
Nathanael/battery fault serial sending

### PR DESCRIPTION
**Pull request by Reddragoneer1**
My other PR is included in this

### CODE REVIEW

**Comment by Reddragoneer1** [comment]
Note: This PR has the left and right switched to match the communications protocol
Also, if we're adding changes to the communications protocol, should we track that somehow either using like a version number or maybe github?

**Comment by bill-luu** [comment]
In the future, if you are continuing or including another PR, close it and mention that it Continues #45

**Comment by bill-luu** [comment]
Also, leave a note in your description saying that it "Fixes #45, Fixes #42". That links the PR to the issues, so when this PR is merged, the issues are closed.

**Comment by bill-luu** [comment]
Google drive lets you track changes to the protocol, so I think that's fine.

File: EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c [hunk 1]
```diff
@@ -201,7 +201,7 @@ void sendBatteryFaults()
 
     packetPayload[0] = BATTERY_FAULTS_PKG_ID;
     writeBoolsIntoArray(packetPayload, 1, &batteryFaultsData.batteryErrorFlags, 21);
-    writeBoolsIntoArray(packetPayload, 5, &batteryFaultsData.batteryLimitFlags, 14);
+    writeBoolsIntoArray(packetPayload, 4, &batteryFaultsData.batteryLimitFlags, 16);
     addChecksum(packetPayload, BATTERY_FAULTS_LENGTH);
```
Comment (enochtsang) [inline EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c]: Is the length of 7 still accurate? otherwise looks ok to me

Comment (Reddragoneer1) [reply EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c]: I think we changed it to 6

Comment (bill-luu) [reply EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c]: Good catch! Yes, please update this to 6.

**Pull request merged**